.evergreen/config/test-variants.yml

```diff
@@ -0,0 +1,134 @@
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-8.2"
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-8.2"
+    tasks:
+      - ".standalone .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - ".replicaset .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - ".sharded .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - "test-atlas-connectivity"
+      - ".ocsp !.4.4"
+    display_tasks:
+      - name: "test-ocsp-latest"
+        execution_tasks:
+          - ".ocsp .latest"
+      - name: "test-ocsp-rapid"
+        execution_tasks:
+          - ".ocsp .rapid"
+      - name: "test-ocsp-7.0"
+        execution_tasks:
+          - ".ocsp .7.0"
+      - name: "test-ocsp-6.0"
+        execution_tasks:
+          - ".ocsp .6.0"
+      - name: "test-ocsp-5.0"
+        execution_tasks:
+          - ".ocsp .5.0"
+  - name: test-debian92-php82-local
+    tags: ["test", "debian", "x64"]
+    display_name: "Test: Debian 9.2, PHP 8.2"
+    run_on: debian92-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian92"
+      FETCH_BUILD_TASK: "build-php-8.2"
+    depends_on:
+      - variant: "build-debian92"
+        name: "build-php-8.2"
+    tasks:
+      # Remember to add new major versions here as they are released
+      - ".standalone .local !.6.0 !.7.0 !.rapid !.latest"
+      - ".replicaset .local !.6.0 !.7.0 !.rapid !.latest"
+      - ".sharded .local !.6.0 !.7.0 !.rapid !.latest"
+      - ".ocsp !.6.0 !.7.0 !.rapid !.latest"
+    display_tasks:
+      - name: "test-ocsp-4.4"
+        execution_tasks:
+          - ".ocsp .4.4"
+
+  # Test remaining PHP versions with replica sets on Debian 11 with MongoDB 7.0
+  - name: test-debian11-php81-local
+    tags: ["test", "debian", "x64"]
+    display_name: "Test: Debian 11, PHP 8.1"
+    run_on: debian11-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-8.1"
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-8.1"
+    tasks:
+      - ".replicaset .local .7.0 .auth"
+  - name: test-debian11-php80-local
+    tags: ["test", "debian", "x64"]
+    display_name: "Test: Debian 11, PHP 8.0"
+    run_on: debian11-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-8.0"
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-8.0"
+    tasks:
+      - ".replicaset .local .7.0 .auth"
+  - name: test-debian11-php74-local
+    tags: ["test", "debian", "x64"]
+    display_name: "Test: Debian 11, PHP 7.4"
+    run_on: debian11-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-7.4"
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-7.4"
+    tasks:
+      - ".replicaset .local .7.0 .auth"
+
+  # Variants with different libmongoc
+  # TODO: Enable the following two builds once libmongoc 1.25.0 is released
```

Comment:
This might be something else to reference from `_template-build-libmongoc.yml`.

Since I expect these bits will be frequently modified, it may be worth calling out in `architecture.md`, or the higher-level `contributing.md` file (assuming it doesn't already talk about enabling and disabling libmongoc versions for Evergreen).

Reply:
Added a note in the contribution docs since that explains the libmongoc update process already.